<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>仪器信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>id:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_id">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>用户单位:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEdit_yonghu"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>客户地址:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_addr">
       <property name="styleSheet">
        <string notr="true">background-color:  rgba(124, 200,128, 255)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>通道配置:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEdit_channels"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>仪器型号:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_yiqixinghao"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>仪器编号:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEdit_yiqibh"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>包箱:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_baoxiang"/>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>审核:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="lineEdit_shenhe"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>预计发货时间:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_yujifahuo_date"/>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>调试时间:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="lineEdit_tiaoshi_date"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>合同编号:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_hetongbh"/>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>方法:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_method">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_deletemethod">
         <property name="text">
          <string>删除方法</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_clear">
       <property name="text">
        <string>清除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_copy">
       <property name="text">
        <string>复制</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="lineEdit_pack">
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="editable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_bibei">
              <property name="text">
               <string>必备</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>新包名称：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_packname"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_newpack">
              <property name="text">
               <string>新包</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_deletepack">
            <property name="text">
             <string>移去</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QTableWidget" name="tableWidget_2"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="pushButton_edititem">
              <property name="text">
               <string>编辑</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_deleteitem">
              <property name="text">
               <string>删除</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>输入已有备件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_item"/>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_2">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLineEdit" name="lineEdit_itemname"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>新备件</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_save</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_clear</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_copy</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_bibei</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>bibei()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>656</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_newpack</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>newpack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>656</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>447</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>Form</receiver>
   <slot>itemselect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_edititem</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>editpackitem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_deleteitem</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>deletepackitem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>651</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>newpackitem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>663</x>
     <y>567</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_deletepack</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>deletepack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>542</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_2</sender>
   <signal>activated(int)</signal>
   <receiver>Form</receiver>
   <slot>selectitem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>708</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>739</x>
     <y>534</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_addr</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>textchange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_yiqixinghao</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>textchange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>745</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_baoxiang</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>textchange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>743</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_yujifahuo_date</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>textchange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>743</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_hetongbh</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>textchange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>740</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_yonghu</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>textchange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>703</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_channels</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>textchange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>672</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>738</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_yiqibh</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>textchange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>737</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_shenhe</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>textchange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>715</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>745</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_tiaoshi_date</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>textchange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>activated(int)</signal>
   <receiver>Form</receiver>
   <slot>selectpack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_item</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>iteminput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>529</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>740</x>
     <y>482</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>save()</slot>
  <slot>add()</slot>
  <slot>method()</slot>
  <slot>clear()</slot>
  <slot>copy()</slot>
  <slot>bibei()</slot>
  <slot>newpack()</slot>
  <slot>editpack()</slot>
  <slot>deletepack()</slot>
  <slot>itemselect()</slot>
  <slot>deletepackitem()</slot>
  <slot>editpackitem()</slot>
  <slot>newpackitem()</slot>
  <slot>iteminput()</slot>
  <slot>packinput()</slot>
  <slot>selectpack()</slot>
  <slot>selectpack2()</slot>
  <slot>selectitem()</slot>
  <slot>textchange()</slot>
 </slots>
</ui>
